web/app/components/DepositWithdraw/BitKapital.jsx

```diff
@@ -39,7 +39,7 @@ class BitKapital extends React.Component {
     _renderDeposits() {
         return <iframe
             style={{width: "100%", border: 0, minHeight: 800}}
-            src={"https://bitkapital.com/kapital.html?u=" + this.props.account.get("name") + `&theme=${SettingsStore.getState().settings.get("themes")}`}
+            src={"https://bitkapital.com/kapital.html?u=" + this.props.account.get("name") + "&theme=" + ${SettingsStore.getState().settings.get("themes")}}
```

Comment:
This code won't work, the `` is ES6 string syntax that lets you use variables inside ${}, so you don't need to add strings with + anymore. What are you trying to fix?